README.md

```diff
@@ -12,7 +12,12 @@ Helper smart contracts to list assets on Aave v3 or change configs. Designed to
 | MIMATIC (MAI) | asset-listing | Aave v3 Fantom    | [MIMATIC Steward](./src/contracts/mimatic/AaveV3FantomMIMATICListingSteward.sol)                 | [Tests](./src/test/MIMATICAaveV3FantomListingByGuardian.t.sol)           | N/A                                                                                                                       |
 | Multiple      | config-change | Aave v3 Harmony   | [Harmony freezing](./src/contracts/harmony-protection/FreezeHarmonyPoolReservesSteward.sol)      | [Tests](./src/test/FreezeAllReservesAaveV3FantomByGuardian.t.sol)        | [0xf202866d9fb6f089587d86d4128e7c8e0fdf94fe](https://explorer.harmony.one/address/0xf202866d9fb6f089587d86d4128e7c8e0fdf94fe)                                                                                                                        |
 | Multiple      | config-change | Aave v3 Fantom    | [Fantom freezing](./src/contracts/fantom-freeze/FreezeFantomPoolReservesSteward.sol)             | [Tests](./src/test/FreezeAllReservesAaveV3HarmonyByGuardian.t.sol)       | [0x1aa435ed226014407fa6b889e9d06c02b1a12af3](https://ftmscan.com/address/0x1aa435ed226014407fa6b889e9d06c02b1a12af3#code)                                                                                                                        |
-| BTC.b         | asset-listing | Aave v3 Avalanche | [BTC.b Steward](./src/contracts/btc.b/AaveV3AvaBTCBListingSteward.sol)                           | [Tests](./src/test/BTCBAaveV3AvaListingByGuardian.t.sol)                 | [0xeee4877a56392c82578df71e8b9270ad8cbabfdc](https://snowtrace.io/address/0xeee4877a56392c82578df71e8b9270ad8cbabfdc#code) |
+| BTC.b         | asset-listing | Aave v3 Avalanche | [BTC.b Steward](./src/contracts/btc.b/AaveV3AvaBTCBListingSteward.sol)                           | [Tests](./src/test/BTCBAaveV3AvaListingByGuardian.t.sol)                 | [0xeee4877a56392c82578df71e8b9270ad8cbabfdc](https://snowtrace.io/address/0xeee4877a56392c82578df71e8b9270ad8cbabfdc#code) |                                                                                                               |
```

Comment:
Doesn't look like new entries to the table are printing correctly

Reply:
fixed :)